### DISCUSSION TITLE
The Emerald Sword

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8292] [Q&A] [answered]
How do you start the Emerald Sword quest chain (Do you have to be in a certain place at a certain time, or is it just whenever you head to area X, the quest starts)?

**Comment by RisingLeaf** [comment, marked as answer]
https://steamcommunity.com/id/Terin7/myworkshopfiles/?section=guides&appid=404410

May be a bit outdated, but the basics should be the same.

**Reply by SultanRTG** [reply]
I can't access it for some reason, can you just say where to start the quest?

**Reply by RisingLeaf** [reply]
Brute force copy pasta, read what you want and be aware of spoilers:

Introduction
Millennia ago, the Sheragi were a species of dragon-like people that existed in part of the space that humans now occupy in Endless Sky. Your pilot may have even learned some thing about them unknowingly. However, beginning in 0.9.13, the pieces are in place to dig into their history and witness first-hand their last days, and to find one of their powerful warships, the Emerald Sword. This guide will help you through the process, including letting you know if you already completed pre-requisites beforehand.
Deep Archaeology
This sub-plot involves working with the rogue archaeologist, Albert Foster, in discovering some hidden truths in The Deep.

You have completed this if your logbook has the following entry:
Returned to Midgard with Albert Foster, the archaeologist, to map the area around the church. The instruments discovered what looked like an artificial singularity. Got chased away by Deep ships before being able to look closer.

Name: Deep Archaeology 1 (Deep Archaeology)
Requirements: Over 15 Combat Experience (Combat Rating 2.7), 1 spare bunk, 15% Random Chance
Source: Paradise Worlds, Near Earth worlds, North worlds, not Vinci in Miaplacidus
Mission: Meet Archaeologist in Vinci in Miaplacidus

Name: Deep Archaeology 2 (Deep Archaeology)
Requirements: Deep Archaeology 2 Complete
Source: Vinci in Miaplacidus
Mission: Take sample from a building on Midgard in Aludra

Name: Deep Archaeology 3 (Deep Archaeology)
Requirements: Deep Archaeology 2 Complete
Source: Midgard in Aludra
Mission: Return with sample to Vinci in Miaplacidus

Name: Deep Archaeology 4 (Deep Archaeology)
Requirements: Deep Archaeology 3 Complete, 4 cargo space, 2 spare bunks
Source: Vinci in Miaplacidus
Mission: Bring Foster, lab tech, and mapping equipment to Midgard in Aludra

Name: Deep Archaeology 5 (Deep Archaeology)
Requirements: Deep Archaeology 4 Complete, 4 cargo space, 2 spare bunks
Source: Midgard in Aludra
Mission: Flee from hostile Deep ships back to Vinci in Miaplacidus
There Might Be Riots
This sub-plot involves transporting the band, There Might Be Riots, throughout human and alien space, before learning of an interesting vision from them.

You have completed this if your logbook has the following entry:
Brought "There Might Be Riots" to the Hai world of Allhome after escaping a swarm of combat drones that disrupted their performance on Pilot. Ulrich spoke of a cloudy star in the Rim where he heard a voice in his head after parking his ship.

Name: There Might Be Riots 1 (Transport band to Wayfarer)
Requirements: 4 Cargo Space, 8 Spare Bunks, 15% Random Chance
Source: Republic/Free Worlds planet 5-10 jumps from Tarazed
Mission: Bring band to Wayfarer in Tarazed

Name: There Might Be Riots 2 (Transport band to Prime)
Requirements: There Might Be Riots 1 Complete, Deep Archaeology 5 Complete, 4 Cargo Space, 8 Spare Bunks, 30% Random Chance
Source: Syndicate planet
Mission: Bring band to Prime in Betelgeuse

Name: There Might Be Riots 3A (Transport Band to Pilot)
Requirements: There Be Might Riots 2 Complete, First Contact made with Hai, 4 Cargo Space, 8 Spare Bunks, 30% Random Chance
Source: Dirt Belt, Rim, or South planet
Mission: Bring band to Pilot in Elnath

Name: There Might Be Riots 3B (Transport Band to Allhome)
Requirements: There Be Might Riots 3A Complete, 4 Cargo Space, 8 Spare Bunks
Source: Pilot in Elnath
Mission: Flee from Navy drone combat drill, to Allhome in Hai-home
Sad Archie
This here, is the linchpin to finding the Emerald Sword. Prompted by a story of one of the band members of There Might Be Riots receiving a vision while drifting around a "cloudy star out along the Rim.", you must fly to the Ildaria system.

Once there, bribe any pirates nearby (or just have a very strong ship with heavy anti-missile), come to a stop in the center of the system (shift+down makes this very easy), and wait. This will take 30-60 seconds while fast-forwarding, then you will receive a vision. Afterwards, go land in Zug in Zubeneschamali.

Name: Sad Archie
Requirements: There Might Be Riots 3B Complete
Source: Idling in Ildaria
Mission: Land on Zug in Zubeneschamali to confirm your vision
Rim Archaeology
After having experienced a strange vision around Ildaria, and confirmed parts of it with locations on Zug, you notify your archaeologist contact, Albert Foster, that there might be something worth looking into there.

You have completed this if you have the Sheragi entry under Factions in your logbook, with this entry:
Recent archaeological discoveries on Zug, a planet in the southern rim of human space, have turned up evidence of a lost alien civilization. The Quarg say that the ruins were left behind by a species called the Sheragi.

Name: Rim Archaeology 1 (Archaeology on Zug)
Requirements: Sad Archie Complete, 1 Spare Bunk
Source: Vinci in Miaplacidus
Mission: Bring Albert Foster to Zug in Zubeneschamali

Name: Rim Archaeology 2A (Archaeological Equipment)
Requirements: Rim Archaeology 1 Complete
Source: Zug in Zubeneschamali
Mission: Retrieve excavation equipment from Crossroads in Ruchbah

Name: Rim Archaeology 2B (Archaeological Equipment)
Requirements: Rim Archaeology 2A Complete, 28 Cargo Space
Source: Crossroads in Ruchbah
Mission: Deliver excavation equipment to Zug in Zubeneschamali

Name: Rim Archaeology 3A (Archaeological Equipment)
Requirements: Rim Archaeology 2B Complete
Source: Zug in Zubeneschamali
Mission: Retrieve excavation equipment on Asgard in Naos

Name: Rim Archaeology 3B (Archaeological Equipment)
Requirements: Rim Archaeology 3A Complete, 17 Cargo Space
Source: Asgard in Naos
Mission: Evade the Oxyrhynchus and get back to Zug in Zubeneschamali

Name: Rim Archaeology 4A (Transport a Xenobiologist)
Requirements: Rim Archaeology 3B Complete
Source: Zug in Zubeneschamali
Mission: Retrieve a xenobiologist from Lagrange in Enif

Name: Rim Archaeology 4B (Transport a Xenobiologist)
Requirements: Rim Archaeology 4A Complete, 2 Spare Bunks
Source: Lagrange in Enif
Mission: Bring xenobiologist to Zug in Zubeneschamali

Name: Rim Archaeology 5A (Defend Zug)
Requirements: Rim Archaeology 4B Complete
Source: Zug in Zubeneschamali
Mission: Destroy pirates around Zug, or wait for others to destroy them. A Quarg ship is on the way to help.

Name: Rim Archaeology 6
Requirements: Wait 97 days for excavation to complete
Source: Anywhere in human space
Mission: None
The Box
The following mission string is new in v0.9.13. To begin, you must have completed the Free Worlds campaign, made contact with the Unfettered Hai (by landing on one of the hostile Hai worlds in the north part of their space), and pass a 4% random chance on any planet in human space, except for Zug. You could just repeatedly land and take off, but I personally recommend just trying to do other things in human space, and it will naturally appear after time.

Name: The Box 1 (The Sheragi Bunker)
Requirements: Rim Archaeology 6 seen, first contact made with the Unfettered Hai, Free Worlds Complete, 4% Random Chance
Source: Anywhere in human space EXCEPT for Zug
Mission: Return to Zug in Zubeneschamali to see new Sheragi discovery

Name: The Box 2 (The Mysterious Box)
Requirements: The Box 1 Complete
Source: Zug in Zubeneschamali
Mission: Bring cube to Vinci in Miaplacidus

Name: The Box 3 (The Information Dealer)
Requirements: Wait 20-30 days after completing The Box 2
Source: Anywhere in human space
Mission: Find information dealer on Stormhold in Alcyone

Name: The Box 4 (The Search for the Artifact)
Requirements: The Box 3 Complete, Have 10 Million Credits
Source: Stormhold in Alcyone
Mission: Confront mercenaries on Watcher in Algieba

Name: The Box 5 (The Fight for the Artifact)
Requirements: The Box 4 Complete
Source: Watcher in Algieba
Mission: Destroy mercenary fleet around Watcher

Name: The Box 6a/6b (The Box Recovered)
Requirements: The Box 5 Complete
Source: Watcher in Algieba
Mission: Bring the cube back to Vinci in Miaplacidus (Note: You can immediately shoot the mercs to make back almost all the money you'd spend for the information, but there may be consequences in the future)

Name: The Box 6c (Hai Knowledge)
Requirements: The Box 6b Complete
Source: Vinci in Miaplacidus
Mission: Visit Allhome in Heia Due to learn more about the Sheragi

Name: The Box 7a (The Museum of Galactic History)
Requirements: The Box 6c Complete
Source: Allhome in Heia Due
Mission: Visit museum on Mirrorlake in Ya Hai

Name: The Box 7b (The Hai Ministry of Culture)
Requirements: The Box 7a Complete
Source: Mirrorlake in Ya Hai
Mission: Visit Ministry of Culture on Hai-home

Name: The Box 8 (Assembling the Team)
Requirements: The Box 7b Complete, 1 Cargo Space, 3 Spare Bunks
Source: Hai-home in Ya Hai
Mission: Bring Hai archaeologists to Zug in Zubeneschamali
The Emerald Sword
Once you have fully secured the Sheragi black box, and brought in help from the Hai, you're almost done! You will need to wait 40-50 days for them to decode the cube, at which point a new system will be in sight...

Name: The Emerald Sword 1 (Black Box Decoded)
Requirements: Wait 40-50 days after The Box 8
Source: Anywhere in human space except Zug
Mission: Visit Albert Foster at Zug in Zubeneschamali

Name: The Emerald Sword 2 (The Emerald Sword)
Requirements: The Emerald Sword 1 Complete
Source: Zug in Zubeneschamali
Mission: Bring a spare jump drive to Zug, along with a ship with 150 spare bunks and 50 tons of cargo space. You can buy a Dreadnought on Zug, sell all weapons, the Breeder Reactor, and shield generator, and replace those with bunk rooms for enough space, then sell the ship afterwards for all your money back.

Name: The Emerald Sword 3 (The Emerald Sword)
Requirements: The Emerald Sword 2 Complete, 2 Jump Drives, 50 Cargo Space, 150 Spare Bunks
Source: Zug in Zubeneschamali
Mission: Jump to Zubenelhakrabi, repair the Emerald Sword, and land on the Valley of the Damned

Name: Sheragi Archaeology: Epilogue
Requirements: The Emerald Sword 3 Complete, 25 Cargo Space
Source: Valley of the Damned in Zubenelhakrabi
Mission: Return with the Emerald Sword to Zug in Zubeneschamali
Conclusion
You now have access to the Emerald Sword, and its complement of 6 Black Diamond fighters! This ship is filled with unique, hybrid outfits like the Fusion Drive, which includes a reactor with powerful engines and reverse thrusters, or Hybrid Cooling, which uses fuel tanks as passive and active cooling for the ship. The core of the ship, the Dragonflame Cannon, is a truly impressive weapon that outranges most other ships in the game, piercing through shields and delivering loads of heat, while having a heavy energy and heat cost at the same time. With careful use, you can devastate any single target handily.

Your Black Diamond fighters are also unique in having some hybrid outfits, but also their Shard Fabricator weapon. This is a micro-missile launcher, which generates ammo by consuming fuel. This gives these fighters a powerful edge, as long as you can keep them refueled. They will fill up on fuel when launching from your ship, so if you're running low, they will not launch! Make sure to invest in ramscoops or fuel tanks, and be careful, since these fighters cannot be replaced!

As Desyo said when returning with the ship, the Emerald Sword is not invincible. The hull very sturdy, but the shielding is outpaced by some human technology. You may want to experiment with different ship loadouts. When at an outfitter, you can press U on an outfit to permanently store it, since these are entirely unique outfits. Then, you can try different combinations, like Systems Cores to repair hull along with shields, ramscoops to fuel the Dragonflame Cannon, or bunks to have a boarding ship that can capture almost anything.

Experiment to see what you can fill it with, or use it stock, and have fun!

**Comment by Naf5000** [comment]
Strictly speaking, it starts on a northern paradise world near Earth once your combat rating is borderline competent or higher (it can technically be started with lower rating, the way it's calculated is a bit odd) and your ship has enough space for at least one passenger. A man in an expensive suit approaches you for help. The thing is that it's not a 'continuous' questline, there are long breaks in-between missions and some of the triggers aren't particularly clear. Depending on where you are along the line, you may need to land on Vinci for the next part, or a world in the Deep, or have to find They Might Be Riots, or just fly around in the Zug system doing not much for a few minutes. Does the name Albert Foster mean anything to you?

**Reply by SultanRTG** [reply]
Yeah, I did one They Might Be Riots mission and the Deep Samples one.
